lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -871,6 +873,14 @@ def _post(self, data):
         url = reverse('verify_student_create_order')
```

Comment:
This test class is named `TestCreateOrder` and the tests it contains are described as "Tests for the create order view."  Should this be updated? Is the URL being POSTed to here still correct?